spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -245,6 +245,7 @@ def assert_ems
   def assert_specific_container
     @container = Container.find_by(:name => "deployer")
     expect(@container).to have_attributes(
+      :type          => "ManageIQ::Providers::Kubernetes::ContainerManager::Container",
```

Comment:
I would have expected this to be type ManageIQ::Providers::**Openshift**::ContainerManager::Container

Reply:
Exactly :) but we can't do that without https://github.com/ManageIQ/manageiq/pull/20756

Today all Openshift inventory that is shared with kubernetes actually has the kubernetes STI type

Reply:
Ohhhh now i see what you are doing.  That's gross.  Glad you are getting this sorted out!